mithril-common/src/lib.rs

```diff
@@ -10,6 +10,16 @@
 //! - A [certificate chain] used to validate the Certificate Chain created by an aggregator
 //! - The [entities] used by, and exchanged between, the aggregator, signers and client.
 
+macro_rules! cfg_fs {
+    ($($item:item)*) => {
+        $(
+            #[cfg(feature = "fs")]
+            #[cfg_attr(docsrs, doc(cfg(feature = "fs")))]
+            $item
+        )*
+    }
+}
+
```

Comment:
why is this required?

Reply:
due to wasm [build](https://github.com/input-output-hk/mithril/actions/runs/7422372066/job/20197520073?pr=1403) errors, the mithril team suggested a conditional compilation macro.

more details here: https://github.com/input-output-hk/mithril/pull/1403/files/acccd103df4ad21cd126de6e14003135da7105b0#r1442959628